src/peerstate.rs

```diff
@@ -418,7 +417,6 @@ impl<'a> Peerstate<'a> {
                     &self.addr,
                 ],
             )?;
-            reset_gossiped_timestamp(self.context, 0)?;
```

Comment:
I removed this line and an option to pass `0` as `chat_id` to `set_gossiped_timestamp` as resetting gossip on all chats at the same time is an overkill in all cases. Maybe it makes sense to reset gossip timestamps on all chats where some key has been changed, but it should be implemented differently, preferrably outside `peerstate.rs` and may have negative effects, such as rapid propagation of *wrong* key across chats.

Reply:
so we never gossip changed keys directly, only after the 48 hours timeout.

so eg. Alice knows Bob has a new key and Alice knows she told Claire the now wrong key for Bob. but Alice does not correct this. not sure if this is okay, might be, cc @hpk42

Reply:
i guess it's ok.  One practical remedy to a key (change) problem for a contact is, anyway, to have that contact send a message -- all devices in the chat will then see the key update directly, no gossip needed.   Eventually, we want to re-gossip specifically those keys that changed (note that autocrypt gossip does not require sending keys of all chat members, only those keys where the sender thinks it's useful for the receivers to get them).  One problem with gossip is that we never know the timestamp of when the key was directly received -- so when we see diverging gossipped keys we never know which one to prefer.  For directly incoming autocrypt keys from a contact we use the Date as timestamp, and that is properly tracked through the autocrypt logic.  gossip keys are inherently inprecisely tracked (for privacy reasons) and thus it is probably not of much use gossipping more often -- we might even be gossipping the wrong key more often which rather leads to more confusion. 

So for now, the 48h full resend and when adding a new member should suffice to distribute keys and help to ensure encryption consistency, in a data minimizing way.

Reply:
k, sounds reasonable, still not 100% sure - but let's give it a try. and the new simplicity is convincing :)